### PR TITLE
Example handling added

### PR DESCRIPTION
**Pull request by sslb**
Hi,

I've realised that examples in RAML file were ignored, so I added support for them. Seems like you added some code to support parameter examples, but later on they weren't added to the structure (probably just an oversight).

Szabolcs

### CODE REVIEW

**Comment by esh-b** [comment]
Oh, thanks! And yes, when I started, I wrote the code so that the important things (endpoints, methods, params) are taken care of in the first place and the conversion is correct. Thought of doing the minor things later.
So, did you check it's working and its correctness after conversion?

**Comment by sslb** [comment]
Yes, naturally :)

**Comment by esh-b** [comment]
Thanks! I am confirming it again to make sure that the correctness isn't comprised.

File: raml2swagger-lib/src/main/java/org/macrulez/utils/raml2swagger/RAMLtoSwagger.java [hunk 1]
```diff
@@ -635,7 +641,23 @@ private JSONObject getResponseInfo(Map.Entry<ActionType, Action> action) throws
                         fields.put(SCHEMA_PARAM_KEY, jsonObj);
                     }
                 }
+
+                if (example != null && example.length() > 0) {
+                    // If example is JSON, let's format it
+                    if (MIMETYPE_JSON.equals(me.getKey())) {
+                        JSONObject jsonObj = new JSONObject(example);
```
Comment (mikeabrudan) [inline raml2swagger-lib/src/main/java/org/macrulez/utils/raml2swagger/RAMLtoSwagger.java:648]: I would use `new JSONTokener(example).nextValue()` to accommodate JSON arrays too.

Comment (esh-b) [reply raml2swagger-lib/src/main/java/org/macrulez/utils/raml2swagger/RAMLtoSwagger.java:648]: Hey Mike, thanks for the suggestion.
@sslb, could you look into that part of implementation and make the change if that's feasible?

**Pull request merged**